extensions/amp-fit-text/0.1/amp-fit-text.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
Just confirming: are we doing this or are we adding the new doc to `1.0` dir?

Reply:
@sebastianbenz please advise on the appropriate file structure here :)

Reply:
This is correct. The experimental flag will make sure that the bento docs are not surfaced as prod.